src/core/render/compiler.js

```diff
@@ -282,7 +282,7 @@ export class Compiler {
       }
 
       const tree = this.cacheTree[currentPath] || genTree(toc, level);
-      html = treeTpl(tree, /* html */ `<ul>{inner}</ul>`);
+      html = treeTpl(tree, /* html */ '<ul>{inner}</ul>');
```

Comment:
This one is interesting. There's a new sentiment floating around that preferring back ticks is nicer because then you can easily add interpolations at any time without having to manually convert the quotes.

Reply:
It's an interesting position that some devs have been proposing for a long time. Here's an issue in the Prettier issue from way back in 2017: https://github.com/prettier/prettier/issues/54

My position is that we had single quotes before this PR, this PR does not change that, and so we should therefore not allow a potential debate about a lint configuration change nobody is asking for hold up this PR.

Worth considering:

- The idea of using backticks by default is not new. Neither ESLint or Prettier have adopted this position and those maintainers have undoubtably spent *way* more time thinking about it than anyone here has. If one of those tools adopted backticks by default, I would be more likely to adopt it by default as well. Until then, I'm less likely to do so.
- Presumably the reason to adopt a tool like Prettier is to avoid getting mired down in debates about stylistic opinions like this. Prettier doesn't offer an option to favor backticks over single or double quotes. We can enforce backticks by default with ESLint if we use it to override Prettier.

If people are interested in switching to backticks by default, let's discuss that separately. I don't have strong feelings one way or the other, but I lean towards sticking with what we have (probably just out of familiarity and muscle memory).